modules/openy_features/openy_media/modules/openy_media_image/openy_media_image.install

```diff
@@ -291,4 +291,12 @@ function openy_media_image_update_8012() {
     'entity_browser.browser.images_library_embed',
     'views.view.images_library',
   ]);
+
+  $message = "Image Library entity browser doesn't use 'images_library' view anymore. Custom changes may be lost.";
```

Comment:
can we also add an action item to this message
Imagine I'm a site builder - I received this message - what should I do now?

Reply:
@podarok what action item are you expecting here?

Reply:
@podarok at this moment it is too late to do anything. That's why there are no action items. This is just an info message, mostly for developers. @AndreyMaximov  and @dchubar-fj suggest to keep it as is.

Reply:
Ok, fine. But even for developers we should take care about suggesting action item